test/Benchmarks/mlp-bf16-10layers-3584.mlir

```diff
@@ -0,0 +1,213 @@
+// RUN: tpp-opt %s -element-wise-fusion -tile-consumer-and-fuse-producers="tile-sizes=1,1 max-depth=2" |
```

Comment:
`element-wise-fusion` shouldn't be needed anymore with the updated tile-and-fuse pass.